public/js/registerUser.js

```diff
@@ -0,0 +1,44 @@
+const REGISTERERRORMESSAGE = 'Registration has failed. Please refresh the browser and try again'
+
+let sendNewUserDetails = async (data) => {
+
+    let response =  await fetch(`/api/registerUser`,
+        {
+            credentials: "same-origin",
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json',
+            },
+            method: "POST",
+            body: JSON.stringify(data)
+        })
+        .then(data => data.json())
+        .catch(err => {
+            errorMessage(REGISTERERRORMESSAGE, 'message')
+        })
+    return response
+}
+
+let getInputs = () =>{
+    let cleanedEmailInput = encodeURI(document.getElementById('emailNewUser').value)
+    let cleanedPasswordInput = encodeURI(document.getElementById('randomPassword').value)
+    return {'email' : cleanedEmailInput, 'password' : cleanedPasswordInput}
+}
+
+let newUserForm = document.getElementById('registerUserForm')
```

Comment:
do you need to declare variable newUserForm? then use it immediately on next line?

Reply:
I think it just makes the code look cleaner and easier to read, otherwise that line is:
document.getElementById('registerUserForm').addEventListener('submit', async (e) => { //function

Reply:
ok